tools/gen_openvino_dockerfile.py

```diff
@@ -109,8 +179,18 @@ def dockerfile_for_linux(output_file):
     cp ${IPREFIX}/libinference_engine_lp_transformations.so lib/. && \
     cp ${IPREFIX}/libinference_engine_ir_reader.so lib/. && \
     cp ${IPREFIX}/libMKLDNNPlugin.so lib/. && \
-    cp /workspace/install/lib/libngraph.so lib/. && \
-    cp /workspace/openvino/inference-engine/temp/omp/lib/libiomp5.so lib/.
+'''
+
+    if (FLAGS.openvino_version.startswith("2021.02")):
+        df += '''cp /workspace/install/lib/libngraph.so lib/. && \
+    cp /workspace/openvino/inference-engine/temp/omp/lib/libiomp5.so lib/. \
+'''
+    else:
+        df += '''cp /workspace/install/lib/libngraph.so lib/. && \
+    cp /workspace/openvino/inference-engine/temp/omp/lib/libiomp5.so lib/. \
```

Comment:
Seems like nothing is different from 2021.02?

Reply:
My bad.. Fixed it now. I didn't test the variant.